x-pack/plugins/apm/server/lib/backends/get_throughput_charts_for_backend.ts

```diff
@@ -81,7 +78,10 @@ export async function getThroughputChartsForBackend({
     response.aggregations?.timeseries.buckets.map((bucket) => {
       return {
         x: bucket.key + offsetInMs,
-        y: bucket.throughput.value,
+        y: calculateThroughputWithInterval({
+          durationAsSeconds: bucketSize,
+          value: bucket.doc_count,
+        }),
```

Comment:
Can we align this with whatever naming we come up with for `bucketSize`:

```suggestion
        y: calculateThroughputWithInterval({
          bucketSize: bucketSize,
          value: bucket.doc_count,
        }),
```